src/components/ActionButton.jsx

```diff
@@ -0,0 +1,33 @@
+import Button from '@mui/material/Button'
+import { useTheme } from '@mui/material/styles'
+
+//ActionButton receives 'buttonLink and buttonText' from index.js as props.
+//'buttonText' recieves the text you want to place inside of the button.
+//buttonLink determines the href that the user is directed to after clicking the button.
+const ActionButton = ({ buttonLink, buttonText }) => {
```

Comment:
No not a problem necessarily, just when you're using the ActionButton it'll look like:

```jsx
<ActionButton buttonLink="link" buttonText="text" />
```

So the button prefix just seemed redundant since those props are clearly being passed to the button

```jsx
<ActionButton link="link" text="text" />
```